frontend/src/components/VariantListPage/VariantListVariants.tsx

```diff
@@ -131,6 +190,26 @@ const VariantListVariants = (props: VariantListVariantsProps) => {
         {notIncludedVariants.size} variant
         {notIncludedVariants.size !== 1 ? "s" : ""} are not included.
       </Text>
+      <Box mb={4}>
+        <Text>Tagged Groups</Text>
+        <Text>
+          This variant list contains {tagCounts.A} variant
+          {tagCounts.A !== 1 ? "s" : ""} tagged {taggedGroups.A.displayName}.
+        </Text>
+        <Text>
+          This variant list contains {tagCounts.B} variant
+          {tagCounts.B !== 1 ? "s" : ""} tagged {taggedGroups.B.displayName}.
+        </Text>
+        <Text>
+          This variant list contains {tagCounts.C} variant
+          {tagCounts.C !== 1 ? "s" : ""} tagged {taggedGroups.C.displayName}.
+        </Text>
+        <Text>
+          This variant list contains {tagCounts.D} variant
+          {tagCounts.D !== 1 ? "s" : ""} tagged {taggedGroups.D.displayName}.
```

Comment:
All of these could use iteration if you wanted, but some are so few lines I might not bother for now, still, it's a good habit to get into.

The one I would consider using iteration on is the tsx making some html starting on line 193, there you could map over the tagged groups and return `<Text>`s. That might have some slightly noticable benefit if we ever change the stylings, text, etc.